src/AltinnCore/Designer/Controllers/DeployController.cs

```diff
@@ -105,7 +105,7 @@ public async Task<IActionResult> StartDeployment(string org, string app)
             string credentials = _configuration["AccessTokenDevOps"];
 
             string result = string.Empty;
-            Branch masterBranch = _giteaAPI.GetBranch(org, app, "master").Result;
+            Branch masterBranch = null; //_giteaAPI.GetBranch(org, app, "master").Result;
```

Comment:
Intentionally left out?

Reply:
Well, yes, I forgot about that one actually, however the entire file could and should be deleted. So, I'll delete this file now. Thanks for noticing :)